SaintCoinach/ex.json

```diff
@@ -670,6 +679,23 @@
         {
           "index": 21,
           "name": "Aetherstream{Y}"
+        },
+        {
+          "index": 22,
+          "name": "Aetherstream{Z}"
```

Comment:
This may be a sort key, not a Z coordinate.  AetherstreamX/Y are used to calculate teleport costs.